### PR TITLE
[docs] Remove ref to keypairs

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
Remove references to keypairs in the docs.

/cc @aravindhp @sebsoto

### CODE REVIEW

File: tools/windows-node-installer/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Available Commands:
 ```bash
 ./wni aws create --kubeconfig <path to OpenShift cluster>/kubeconfig --credentials <path to aws>/credentials 
 --credential-account default --image-id ami-06a4e829b8bbad61e --instance-type m4.large --ssh-key <name of the 
-existing ssh key, ie: libra> --private-key <private key to decrypt the aws instance password>
+existing ssh keypair in aws> --private-key <private key to decrypt the aws instance password.>
```
Comment (aravindhp) [inline tools/windows-node-installer/README.md]: Nit: `.` is not needed within `<>`

**Comment by ravisantoshgudimetla** [comment]
@sebsoto why hold? is it just for nit?

**Comment by sebsoto** [comment]
Yeah, just for nit, didnt want it to get automatically merged. feel free to unhold

**Pull request merged**